h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -312,8 +332,8 @@ def test_no_time_limit_if_max_models_is_provided():
     aml.train(y=ds['target'], training_frame=ds['train'])
     max_runtime = aml._build_resp['build_control']['stopping_criteria']['max_runtime_secs']
     max_models = aml._build_resp['build_control']['stopping_criteria']['max_models']
-    assert max_runtime == 0
-    assert max_models == 1
+    assert max_models == 1, max_models
```

Comment:
@sebhrusen have not seen this type of syntax. Could you please share with me what does it mean when we add comma after comparison?

Reply:
it's just the message printed on failed assertion

Reply:
https://docs.python.org/3/reference/simple_stmts.html#the-assert-statement

Reply:
@sebhrusen thanks, this was my number one guess, just usually Failing messages contain some text for context.

Reply:
Python error also prints the code line raising the assertion, so it's not as necessary as in Java to be verbose in description